docs/src/content/docs/commands/approvals/index.md

```
@@ -0,0 +1,56 @@
+|---------|-------------|-------------|
+| `/approval` | No description available | ❌ |
+| `/approve` | List all approved users | ❌ |
+| `/approved` | List all approved users | ❌ |
+| `/unapprove` | Remove all approvals (creator only) | ❌ |
+| `/unapproveall` | Remove all approvals (creator only) | ❌ |
+
+## Usage Examples
+
+### Basic Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)